src/cuitensor.jl

```diff
@@ -28,7 +28,7 @@ cuITensor(A::ITensor) = store(tensor(A)) isa ITensors.Empty ? cuITensor(A.inds)
 cu(A::ITensor) = cuITensor(A)
 
 function Base.collect(A::ITensor)
```

Comment:
I'm confused about this definition, is this meant to transfer from CPU to GPU? I thought collect always returned a `Vector`.

Reply:
It transfers from GPU to CPU. `collect` has been used this way in CUDA.jl.

Reply:
Gotcha. What about a function like `cpu` for this? The meaning of that would be clearer to me.

Reply:
We can swap to that, I can make a new PR for it after this.

Reply:
Ok, thanks!